products/jbrowse-desktop/package.json

```diff
@@ -69,14 +68,16 @@
     "cra-start": "rescripts start",
     "test": "rescripts test",
     "build": "rescripts build",
-    "build-electron": "yarn build && electron-builder build --windows --mac --linux",
-    "build-electron:win": "yarn build && electron-builder build --windows",
-    "build-electron:mac": "yarn build && electron-builder build --mac",
-    "build-electron:linux": "yarn build && electron-builder build --linux",
-    "build-electron-unpacked": "yarn build && electron-builder build --windows --mac --linux --dir",
-    "build-electron-unpacked:win": "yarn build && electron-builder build --windows --dir",
-    "build-electron-unpacked:mac": "yarn build && electron-builder build --mac --dir",
-    "build-electron-unpacked:linux": "yarn build && electron-builder build --linux --dir",
+    "prebuild-electron": "yarn build",
```

Comment:
This only runs before `build-electron` and not e.g. `build-electron:linux`

Reply:
darn thought i was being clever. reverted